### PR TITLE
MultiversX LLM fixes

### PR DESCRIPTION
**Pull request by AlexandruPislariu**
### 📝 Description

MultiversX LLM fixes

### ❓ Context

- **Impacted projects**: `` 
- **Linked resource(s)**: `` 

### ✅ Checklist

- [ ] **Test coverage** 
- [ ] **Atomic delivery** 
- [ ] **No breaking changes** 

### 🚀 Expectations to reach

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

### CODE REVIEW

File: apps/ledger-live-mobile/src/families/elrond/components/Details/Details.tsx [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+      }
+    },
+    [account],
+  );
+
+  /*
+   * Return the following fragment if the operation type is delegation.
+   */
+
+  if (type === "DELEGATE") {
```
Comment (henrily-ledger) [inline apps/ledger-live-mobile/src/families/elrond/components/Details/Details.tsx:72]: probably better to do a switch(type) {
case :
}

File: apps/ledger-live-mobile/src/families/elrond/components/Details/index.ts [hunk 2]
```diff
@@ -0,0 +1,6 @@
+import OperationDetailsExtra from "./Details";
+
+import { getURLFeesInfo } from "./helpers/getURLFeesInfo";
+import { getURLWhatIsThis } from "./helpers/getURLWhatIsThis";
+
+export default { getURLWhatIsThis, getURLFeesInfo, OperationDetailsExtra };
```
Comment (henrily-ledger) [inline apps/ledger-live-mobile/src/families/elrond/components/Details/index.ts:6]: getURLWhatIsThis, getURLFeesInfo

I can't find them use anywhere on code ? Did I miss something ?

File: package.json [hunk 3]
```diff
@@ -100,7 +100,7 @@
     "migrate-branch": "workspace:*",
     "pnpm-utils": "workspace:*",
     "rimraf": "^3.0.2",
-    "turbo": "^1.6.1",
+    "turbo": "^1.6.3",
```
Comment (henrily-ledger) [inline package.json:103]: Is it possible to revert it, I guess we don't want to put a change on this version yet or we will do it on our side

File: apps/ledger-live-mobile/src/families/elrond/components/Body/components/Drawer/Drawer.tsx [hunk 4]
```diff
@@ -45,15 +45,15 @@ const Drawer = (props: DrawerPropsType) => {
       actions={actions}
       account={account}
       data={items}
-      ValidatorImage={({ size }) => (
```
Comment (henrily-ledger) [inline apps/ledger-live-mobile/src/families/elrond/components/Body/components/Drawer/Drawer.tsx:48]: I prefer the {{ size }} writing than image.size

File: apps/ledger-live-mobile/src/families/elrond/components/Details/index.ts (hunk 2 above)
Comment (haammar-ledger) [reply apps/ledger-live-mobile/src/families/elrond/components/Details/index.ts:6]: Used here it seems
apps/ledger-live-mobile/src/screens/OperationDetails/index.tsx
apps/ledger-live-mobile/src/screens/OperationDetails/Content.tsx

File: apps/ledger-live-mobile/src/families/elrond/components/Body/components/Delegations/Delegations.tsx [hunk 5]
```diff
@@ -123,7 +123,7 @@ const Delegations = (props: DelegationsPropsType) => {
    * Return an introductory panel to delegations if the user hasn't previously delegated.
    */
 
-  if (delegatedAmount.eq(0) && rewardsAmount.eq(0)) {
+  if (delegatedAmount.isEqualTo(0) && rewardsAmount.isEqualTo(0)) {
```
[the remark this answers is not in the document]
Comment (MiroMargineanu) [reply apps/ledger-live-mobile/src/families/elrond/components/Body/components/Delegations/Delegations.tsx:126]: I've actually changed them for better readability when going through the code. I can revert if necessary?

[the remark this answers is not in the document]
Comment (haammar-ledger) [reply apps/ledger-live-mobile/src/families/elrond/components/Body/components/Delegations/Delegations.tsx:126]: Nope that's fine :+1: thanks for clarifying.

File: apps/ledger-live-mobile/src/families/elrond/components/Body/components/Drawer/Drawer.tsx (hunk 4 above)
Comment (MiroMargineanu) [reply apps/ledger-live-mobile/src/families/elrond/components/Body/components/Drawer/Drawer.tsx]: Changed. 👍

File: apps/ledger-live-mobile/src/families/elrond/components/Body/components/Delegations/Delegations.tsx (hunk 5 above)
[the remark this answers is not in the document]
Comment (AlexandruPislariu) [reply apps/ledger-live-mobile/src/families/elrond/components/Body/components/Delegations/Delegations.tsx:126]: The difference is that .eq() returns a boolean and .isEqualTo() returns a BigNumber. 
We will switch back to .eq() since it is more appropiate in our case.

File: libs/ledger-live-common/src/families/elrond/__snapshots__/bridge.integration.test.ts.snap [hunk 6]
```diff
@@ -20,12 +20,12 @@ Array [
...
     "id": "js:2:elrond:erd1vgfp3g7azqjx4wsmtt7067m0l62v3psmqzr24j6xvywj2tlz0gesvyzsq2:",
     "index": 0,
-    "name": "Elrond 1",
+    "name": "MultiversX 1",
     "nfts": undefined,
     "operationsCount": 6,
     "pendingOperations": Array [],
     "seedIdentifier": "",
-    "spendableBalance": "175162251239346240",
+    "spendableBalance": "180162251239346240",
```
Comment (haammar-ledger) [inline libs/ledger-live-common/src/families/elrond/__snapshots__/bridge.integration.test.ts.snap:28]: Wondering why the balance changed here, does that indicate the account isn't frozen?

File: libs/ledger-live-common/src/families/elrond/js-getTransactionStatus.ts [hunk 7]
```diff
@@ -134,9 +134,10 @@ const getTransactionStatus = async (
   }
 
   if (!errors.amount && totalSpentEgld.gt(a.spendableBalance)) {
-    errors.amount = tokenAccount
-      ? new NotEnoughBalanceInParentAccount()
-      : new NotEnoughBalance();
+    errors.amount =
+      tokenAccount || !["delegate", "send"].includes(t.mode)
```
Comment (haammar-ledger) [inline libs/ledger-live-common/src/families/elrond/js-getTransactionStatus.ts:138]: Are you sure about this? Means "_if the tx is neither a delegate or a send, use the `NotEnoughBalanceInParentAccount` error_", but it doesn't seem related at all?

File: libs/ledger-live-common/src/families/elrond/js-synchronisation.ts [hunk 8]
```diff
@@ -9,10 +9,29 @@ import {
 } from "./api";
 import elrondBuildESDTTokenAccounts from "./js-buildSubAccounts";
 import { reconciliateSubAccounts } from "./js-reconciliation";
-import { FEES_BALANCE } from "./constants";
-import { TokenAccount } from "@ledgerhq/types-live";
+import { Operation, SubAccount, TokenAccount } from "@ledgerhq/types-live";
 import { computeDelegationBalance } from "./logic";
-import BigNumber from "bignumber.js";
+
+function pruneOperations(
```
Comment (haammar-ledger) [inline libs/ledger-live-common/src/families/elrond/js-synchronisation.ts:15]: Could you please explain why there is a need to prune operations? My concern is to understand why we have duplicate operations in the first place.
Didn't have time to look into that area, but I was suspecting more the `reconciliateSubAccounts` part, did you look in that direction?

File: libs/ledger-live-common/src/families/elrond/js-synchronisation.ts [hunk 9]
```diff
@@ -63,17 +82,15 @@ const getAccountShape: GetAccountShape = async (info, syncConfig) => {
   return {
     id: accountId,
     balance: balance.plus(delegationBalance),
-    spendableBalance: balance.gt(FEES_BALANCE)
-      ? balance.minus(FEES_BALANCE)
-      : new BigNumber(0),
+    spendableBalance: balance,
```
Comment (haammar-ledger) [inline libs/ledger-live-common/src/families/elrond/js-synchronisation.ts:85]: Any reason for that change? Can't `spendableBalance` be negative?

Comment (AlexandruPislariu) [reply libs/ledger-live-common/src/families/elrond/js-synchronisation.ts:85]: This is also related to FEES_BALANCE which is not kept anymore.

File: libs/ledger-live-common/src/families/elrond/js-getTransactionStatus.ts (hunk 7 above)
Comment (AlexandruPislariu) [reply libs/ledger-live-common/src/families/elrond/js-getTransactionStatus.ts:138]: I think we need another error here, something like "Not enough EGLD to pay the transaction fees".

Comment (AlexandruPislariu) [reply libs/ledger-live-common/src/families/elrond/js-getTransactionStatus.ts:138]: Done

File: libs/ledger-live-common/src/families/elrond/__snapshots__/bridge.integration.test.ts.snap (hunk 6 above)
Comment (AlexandruPislariu) [reply libs/ledger-live-common/src/families/elrond/__snapshots__/bridge.integration.test.ts.snap:28]: This means that users can now spend all their balance.

File: libs/ledger-live-common/src/families/elrond/js-synchronisation.ts (hunk 8 above)
Comment (haammar-ledger) [reply libs/ledger-live-common/src/families/elrond/js-synchronisation.ts:15]: Got it, ok if the duplication originates from the backend itself then frontend has to deduplicate, makes sense. Thanks.

**Review by haammar-ledger — APPROVED** [review]
LGTM, thanks guys!
